packages/service/src/lib/command.ts

```diff
@@ -41,9 +46,16 @@ const getProperties = (obj: any): string[] => {
   return result;
 };
 
+const getOptions = (data: any): IRunOptions => {
+  const silent = !!(data && data.parent && data.parent.silent);
+  const debug = !!(data && data.parent && data.parent.debug);
```

Comment:
I still didn't grasp the whole setup, maybe would be a good idea to type `data` ? 

what is parent in that `context`?

Reply:
data is not really type-able because the underlying cli-package is not typed and always has a different structure, so `any` is the easiest option. I will iterate once more on this PR 👍